app/models/data_update_script.rb

```diff
@@ -8,17 +8,23 @@ class DataUpdateScript < ApplicationRecord
   enum status: STATUSES
   validates :file_name, uniqueness: true
 
-  def self.filenames
-    Dir.glob("*.rb", base: DIRECTORY).map do |f|
-      Pathname.new(f).basename(".rb").to_s
+  class << self
```

Comment:
I liked it better without `class << self` tbh. Personal preference though, so don't let that stop you. 

I just generally favor the slightly more verbose explicit forms. I.e. since `def` returns the method name as symbols, you can actually use access modifiers like this:

```ruby
private def foo
  ...
end
```

I used to work at a company where we used this style by default, and it was great because it allowed you to keep related things together, which the normal access modifiers don't.

Reply:
I think sooner rather than later we'll need to start writing down some style guides as Rubocop has no opinions on this things I guess. I like `class << self` instead of the other style, but yeah we should have a consensus. Debating over style should only be done once while writing down which for a shared style guide :) Maybe we can start working on that in the next iteration.

Reply:
No debate, just starting to elicit opinions on different things 😃 I can live with any styles as long as the team agrees. I'm just bringing things up so we do actually reach a consensus on them, so 👍 for at least getting the ball rolling on this in the next iteration.